tests/integration_tests/functional/test_mmds.py

```diff
@@ -116,6 +118,31 @@ def test_custom_ipv4(test_microvm_with_ssh, network_config):
     _, stdout, _ = ssh_connection.execute_command(cmd)
     assert json.load(stdout) == data_store['latest']['meta-data']
 
+@pytest.mark.timeout(1200)
+def test_big_buffer(test_microvm_with_ssh, network_config):
+    """
+    Test send large buffer.
```

Comment:
we should have a more descriptive name of the function and description. It'll appear in the test report and the test scenario is quite ambiguous